docs/detections/rules-ui-manage.asciidoc

```diff
@@ -104,6 +105,25 @@ NOTE: When duplicating a rule with exceptions, you can choose to duplicate the r
 * To enable or disable a single rule, switch on the rule's *Enabled* toggle.
 * To <<snooze-rule-actions,snooze>> actions for rules, click the bell icon.
 
+[float]
+[[manually-run-rules]]
+=== Run rules manually
+
+Manually run enabled rules for a specified period of time for testing purposes or additional rule coverage. 
+
+IMPORTANT: Before manually running rules, make sure you properly understand and plan for rule dependencies. Incorrect scheduling can lead to inconsistent rule results.
+
+. Go to **Rules** -> **Detection rules (SIEM)**.
+. Do one of the following:
+* Select the **All actions** menu on a rule (**...** → **All actions** in the rules list), then select **Manual run**.
+* Select all the rules you want to manually run, select the **Bulk actions** menu, then select **Manual run**.
+. Specify when the manual run starts and ends. The default selection is the current day starting three hours in the past. The rule will search for events during the selected time range.
+. Click **Run** to manually run the rule.
+
+The manual run's details are recorded in the <<manual-runs-table,Manual runs>> table on the *Execution results* tab. Changes you make to the manual run or rule settings will display in the Manual runs table after the current run completes.
+
+NOTE: Manual runs are executed with low priority and limited concurrency, meaning they might take longer to complete. This can be especially apparent for rules requiring multiple executions.
```

Comment:
```suggestion
[NOTE] 
=====
Note the following:
* Rules actions are not activated during manual runs. 
* Except for threshold rules, duplicate alerts are not created if a rule is manually run during a time range that was already covered by a scheduled run.
* Manual runs are executed with low priority and limited concurrency, meaning they might take longer to complete. This can be especially apparent for rules requiring multiple executions.
=====
```